### PR TITLE
SDL_LockMutex: Added code example.

### PR DESCRIPTION
**Pull request by SDLWikiBot**
This edit was made by @FtZPetruska.

Live page is here: https://wiki.libsdl.org/SDL_LockMutex

If this user should be blocked from further edits, an admin should go to https://wiki.libsdl.org/FtZPetruska/block
If this user should be trusted to make direct pushes to main, without a pull request, an admin should go to https://wiki.libsdl.org/FtZPetruska/trust

### CODE REVIEW

File: SDL_LockMutex.mediawiki [hunk 1]
```diff
@@ -35,7 +35,29 @@ This function is available since SDL 2.0.0.
 
 == Code Examples ==
 
-<<Include(SDL_CreateMutex, , , from="## Begin Mutex Example", to="## End Mutex Example")>>
+<!-- # Begin Mutex Example -->
+<syntaxhighlight lang='c++'>
```
Comment (slouken) [inline SDL_LockMutex.mediawiki:39]: The language should be C

File: SDL_LockMutex.mediawiki [hunk 2]
```diff
@@ -35,7 +35,29 @@ This function is available since SDL 2.0.0.
...
+int status;
+SDL_mutex *mutex;
+
+mutex = SDL_CreateMutex();
+if (!mutex) {
+  fprintf(stderr, "Couldn't create mutex\n");
```
Comment (slouken) [inline SDL_LockMutex.mediawiki:45]: This should be SDL_Log()

File: SDL_LockMutex.mediawiki [hunk 3]
```diff
@@ -35,7 +35,29 @@ This function is available since SDL 2.0.0.
...
+  return;
+}
+
+status = SDL_LockMutex(mutex);
+
+if (status == 0) {
+  printf("Locked mutex\n");
```
Comment (slouken) [inline SDL_LockMutex.mediawiki:52]: This should be SDL_Log()

**Comment by FtZPetruska** [comment]
Since this PR and #196 were auto generated with the `edit` button on the Wiki, it doesn't seem like I have a way to push the requested fixes on these branches due to lack of write access. Moreover, I am not allowed by github to attach patch files either.

Would you like me create a new PR for each that includes the fixes?

**Comment by slouken** [comment]
Yes, we can close the existing PR. Thanks!

**Pull request closed without merging**